helm/thingsboard/templates/node-configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
       export JAVA_OPTS="$JAVA_OPTS -XX:+IgnoreUnrecognizedVMOptions -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/log/thingsboard/${TB_SERVICE_ID}-heapdump.bin"
       export JAVA_OPTS="$JAVA_OPTS -XX:-UseBiasedLocking -XX:+UseTLAB -XX:+ResizeTLAB -XX:+PerfDisableSharedMem -XX:+UseCondCardMark"
       export JAVA_OPTS="$JAVA_OPTS -XX:+UseG1GC -XX:MaxGCPauseMillis=500 -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled -XX:MaxTenuringThreshold=10"
-      export JAVA_OPTS="$JAVA_OPTS -XX:+ExitOnOutOfMemoryError"
+      export JAVA_OPTS="$JAVA_OPTS -XX:+ExitOnOutOfMemoryError -XX:+UseContainerSupport -XX:MaxRAMPercentage=80"
```

Comment:
we could add here too https://github.com/midokura/thingsboard-ce-k8s/blob/026335a33cb4e83fe3913b086968b8fd7b262b8c/helm/thingsboard/templates/transport-configmap.yaml#L34
this should be for the mqtt-transport

Reply:
I'm checking how transport is doing.

Reply:
```
(⎈|evpperf:evp2-tb) ➜ k exec evp-mqtt-transport-0 -- jcmd 1 VM.flags | grep MaxHeapSize                                                  
-XX:-AlwaysTenure -XX:CICompilerCount=3 -XX:ConcGCThreads=1 -XX:+ExitOnOutOfMemoryError -XX:G1ConcRefinementThreads=4 -XX:G1HeapRegionSize=1048576 -XX:GCDrainStackTargetSize=64 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/log/tb-mqtt-transport/tb-mqtt-transport-heapdump.bin -XX:+IgnoreUnrecognizedVMOptions -XX:InitialHeapSize=469762048 -XX:MarkStackSize=4194304 -XX:MaxGCPauseMillis=500 -XX:MaxHeapSize=7516192768 -XX:MaxNewSize=4508876800 -XX:MaxTenuringThreshold=10 -XX:MinHeapDeltaBytes=1048576 -XX:-NeverTenure -XX:NonNMethodCodeHeapSize=5830732 -XX:NonProfiledCodeHeapSize=122913754 -XX:+ParallelRefProcEnabled -XX:+PerfDisableSharedMem -XX:ProfiledCodeHeapSize=122913754 -XX:ReservedCodeCacheSize=251658240 -XX:+ResizeTLAB -XX:+SegmentedCodeCache -XX:-UseBiasedLocking -XX:+UseCompressedClassPointers -XX:+UseCompressedOops -XX:+UseCondCardMark -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+UseTLAB 
(⎈|evpperf:evp2-tb) ➜ k exec evp-mqtt-transport-0 -- java -XX:+UnlockDiagnosticVMOptions -XX:+PrintFlagsFinal -version | grep MaxRAM     
 uint64_t MaxRAM                                   = 137438953472                           {pd product} {default}
    uintx MaxRAMFraction                           = 4                                         {product} {default}
   double MaxRAMPercentage                         = 25.000000                                 {product} {default}
openjdk version "11.0.16" 2022-07-19
OpenJDK Runtime Environment (build 11.0.16+8-post-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Debian-1deb11u1, mixed mode, sharing)
```
According to `-XX:MaxHeapSize=7516192768` we're in the same situation. I will update mqtt transport too to be safe.

Reply:
Done https://github.com/midokura/thingsboard-ce-k8s/pull/20/commits/8d2d3f9574210d3143d75535aacc029326b11390

Reply:
perfect